docs/source/udf.rst

```diff
@@ -14,46 +15,47 @@ How UDF works in layman's terms
 
 The UDF interface of LiberTEM is heavily utilizing the existing LiberTEM architecture. First,
 data is partitioned into several `"partitions"` and distributed across workers. Then, each partition,
-which can be viewed as a collection of frames, are processed by the user-defined `process_frame` method.
-Here, frames are fetched in an iterative manner, and `process_frame` method performs user-defined operations
+which can be viewed as a collection of frames, are processed by the user-defined :meth:`process_frame` or :meth:`process_tile` method. Tiles are described in more detail in `tiled processing`_ and are useful to improve the performance of some numerical operations. In the following, the simpler :meth:`process_frame` interface is described first.
```

Comment:
These are still not linked to the right places in the reference documentation; the proper way appears to be to mention the fully-qualified name and prefix with a tilde, i.e. 

```
:meth:`~libertem.udf.UDFFrameMixin.process_frame`
```

That should properly link to the reference, if one exists.